charts/xui-mo-webapp/values.preview.template.yaml

```diff
@@ -14,8 +14,8 @@ nodejs:
     #SERVICES_TERMS_AND_CONDITIONS_API_SERVICE: http://xui-terms-and-conditions-aat.service.core-compute-aat.internal
     #S2S_SERVICE: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     #FEE_AND_PAY_API: https://payment-api-aat.service.core-compute-aat.internal
-    MANAGE_CASE_LINK: https://xui-webapp-aat.service.core-compute-aat.internal/cases
-    MANAGE_ORG_LINK: https://xui-mo-webapp-aat.service.core-compute-aat.internal/
+    MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/
```

Comment:
should this not be?

MANAGE_CASE_LINK: https://manage-case.aat.platform.hmcts.net/cases